test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BundleEdgeCaseTests.cs

```diff
@@ -31,6 +35,7 @@ public BundleEdgeCaseTests(HttpIntegrationTestFixture fixture)
         }
 
         [Fact]
+        [HttpIntegrationFixtureArgumentSets(dataStores: DataStore.CosmosDb)]
```

Comment:
Why is this one only for Cosmos?

Reply:
That's a good question. I created this test focused on a scenario for Gen1. But it makes sense to keep it running for both generations.